recipes/tidwall-neco/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.microsoft import is_msvc
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.53.0"
+
+class TidwallNecoConan(ConanFile):
+    name = "tidwall-neco"
+    description = "Concurrency library for C (coroutines)"
+    license = "MIT",
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/tidwall/neco"
+    topics = ("coroutine", "concurrency", "network")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", src=self.recipe_folder, dst=self.export_sources_folder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if is_msvc(self):
```

Comment:
I think this is no longer the case in the new version? Could you try to disable this check, see if this works now?

Reply:
@RubenRBS 
Removed.
Let's see the result.

Reply:
@RubenRBS 

There is still compilation errors on MSVC.
It is caused by lack of `stdatomic.h` which is introduced in C11.
Because I have no idea to workaround it, I'm going to drop support for msvc.

```
C:\J\workspace\prod-v1\bsr@4\43174\abeda\.conan\data\tidwall-neco\0.3.2\_\_\build\d057732059ea44a47760900cb5e4855d2bea8714\src\neco.c(163,1): fatal error C1083: Cannot open include file: 'stdatomic.h': No such file or directory [C:\J\workspace\prod-v1\bsr@4\43174\abeda\.conan\data\tidwall-neco\0.3.2\_\_\build\d057732059ea44a47760900cb5e4855d2bea8714\build\neco.vcxproj]
```